### ISSUE TITLE
Improve Performance on Windows

### ISSUE THREAD

**Issue by thursby** [flurinduerst/WPDistillery#78]
Using vagrant-winnfsd provides a huge speed boost to sites that use a lot of files. Load times went from 15-20 seconds per page down to a second or two with the following modifications:

First, `vagrant plugin install vagrant-winnfsd` to install the plugin.

Then I modified the `Vagrantfile` to mount the drive via NFS instead of the built-in Virtualbox provider. 
```ruby
    config.vm.synced_folder ".", "/var/www",
      nfs: true,
      mount_options: [
      'nfsvers=3',
      'vers=3',
      'actimeo=1',
      'rsize=8192',
      'wsize=8192',
      'timeo=14'
      ]
```
This would be a nice feature addition to WP Distillery for any Windows users. It should probably be wrapped in a "if os=windows and vagrant-winnfsd=is_installed" statement (mangled appropriately for Ruby) and would also need the additional documentation created.

Let me know if this is desirable and I'll create a PR.

**Comment by flurinduerst** [comment]
Hey thrusby, thanks a lot for your effort. Sounds like a great improvement for windows users.

20s to 2s sounds like a no brainer, but how much is "a lot of files" as you describe it and how does it change the performance on smaller sites? I can't test this on windows right now.

If  vagrant-winnfsd will improve on more than ~30% of the current windows users, I'd love this addition. If it only helps with less than ~30%, we might just add it as an external "nice to have".

**Comment by thursby** [comment]
I have 8.6k files in my WP installation, and it looks like a stock-ish wpdistillery install contains 1.8k. For what it's worth, the majority of this is from the Jetpack and Mailpoet plugins. Even though these plugins are mostly neutered in my dev environment, it still seems to contribute to overall speed.

To clarify, these load times aren't on the frontend. Instead this is in the admin console, editing posts and previewing, which definitely takes longer than an average page view. I'm developing a plugin that operates on shortcodes so this time is important to me.

Just to make sure I wasn't kidding myself I checked out a new clone of WP Distillery, pulled my test site in, and tested before and after that one modification. To start, I was seeing 12-30 second load times as measured in Chrome's dev console. After making that one modification the longest load time was 6s. On top of that everything just feels faster, so I've re-convinced myself.

To be honest, I can't feel the difference with a vanilla WP install, but nobody stays vanilla for long.

I have created pull request https://github.com/flurinduerst/WPDistillery/pull/79 for your perusal.
